develop/tutorials/articles/192-item-selector/01-selecting-entities-using-the-item-selector-api.markdown

```diff
@@ -0,0 +1,356 @@
+        
+                   on: {
+        
+                      selectedItemChange: function(event) {
+        
+                         var selectedItem = event.newVal;
+        
+        var itemValue = JSON.parse(selectedItem.value);
+        
+        itemSrc =  itemValue.url;
+        
+        [use itemSrc as needed]
+        
+                         }
+        
+                      }
+        
+                   },
+        
+                   title: '<liferay-ui:message key="language-message-key" />',
+        
+                   url: '<%= itemSelectorURL.toString() %>'
+        
+                }
+        
+             );
+        
+             itemSelectorDialog.open();
+        
+          }
+        
+        );
+        
+        </aui:script>
+    
+    For example, using the Java code example in the last section, the logic 
+    would be as follows:
+    
+        <aui:button name="chooseImage" value="Choose" />
+    
+        <aui:script use="liferay-item-selector-dialog">
+        
+        $('#<portlet:namespace />chooseImage').on(
+        
+          'click',
+        
+          function(event) {
+        
+             var itemSelectorDialog = new A.LiferayItemSelectorDialog(
+        
+                {
+        
+                   eventName: 'sampleTestSelectItem',
+        
+                   on: {
+        
+                      selectedItemChange: function(event) {
+        
+                         var selectedItem = event.newVal;
+        
+        var itemValue = JSON.parse(selectedItem.value);
+        
+        itemSrc =  itemValue.url;
+        
+        [use itemSrc as needed]
+        
+                         }
+        
+                      }
+        
+                   },
+        
+                   title: '<liferay-ui:message key="select-image" />',
+        
+                   url: '<%= itemSelectorURL.toString() %>'
+        
+                }
+        
+             );
+        
+             itemSelectorDialog.open();
+        
+          }
+        
+        );
+        
+        </aui:script>
+
+When the user clicks the *Choose* button, a new dialog will pop up, rendering 
+the Item Selector. The `<aui:script>` `use` attribute is set to use the 
+`liferay-item-selector-dialog` module. The trigger is referenced by its ID, 
+using the `$()` selector. Note that the `<portlet:namespace />` tag is prefixed 
+to the ID to avoid name conflicts with other elements on the page. Next, the 
+Item Selector dialog is configured. The example Item Selector configuration 
+renders the views that meet the `ImageItemSelectorCriterion` and returns a URL 
+when the image is selected. Note that the JavaScript event name you created in 
+the last section is used as the `eventName` attribute value, and the 
+`PortletURL` you obtained in the last section is used as the `url` attribute 
+value.
+
+Now you know how to use the Item Selector's APIs to select entities!
```

Comment:
Hey @sergiogonzalez (/cc @mwilliams2014),

Before deciding where to document the tags, are the tags and AlloyUI component interchangeable? When is it better or appropriate to use one of the tags? Or to use the AlloyUI component? - Mike brought up these questions

Thanks.